skribenten-backend/.nais/skribenten-backend-dev.yaml

```diff
@@ -0,0 +1,113 @@
+apiVersion: "nais.io/v1alpha1"
+kind: "Application"
+metadata:
+  name: "skribenten-backend"
+  namespace: "pensjonsbrev"
+  labels:
+    "team": "pensjonsbrev"
+spec:
```

Comment:
Ble vi enige om vi skulle gå tilbake til å ha en felles config for det som er felles igjen?